src/modules/twap/containers/TwapFormWidget/index.tsx

```diff
@@ -47,6 +56,59 @@ export function TwapFormWidget() {
     isCustomDeadline,
   }
 
+  const LABELS_TOOLTIPS: LabelTooltipItems = {
+    numberOfParts: {
+      label: 'No. of parts',
+      tooltip: <>Your TWAP order will be split into this many parts, which will execute one by one.</>,
```

Comment:
Do you need to wrap the tooltip contents in the fragment when it isn't an actual component?

If the property where this is used is of type ReactNode it will accept a component or a raw string

Reply:
`ReactNode` already accepts string or JSX element https://reactnative.dev/docs/react-node
That's what I mean, only ReactNode type should be enough

Reply:
Ah got it! Will update thank you